src/test/java/pages/DynamicControlsPage.java

```diff
@@ -0,0 +1,81 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+
+public class DynamicControlsPage extends BasePage {
+    public static final String DYNAMIC_CONTROLS_URL = "http://the-internet.herokuapp.com/dynamic_controls";
+    public final By CHECKBOX_ELEMENT = By.id("checkbox");
+    public final By BUTTON_ENABLE = By.xpath("/html/body/div[2]/div/div[1]/form[2]/button");
+    public final By FIELD_INPUT = By.xpath("//*[@id='input-example']/input");
+    public final By BUTTON_REMOVE = By.xpath("//*[@id='checkbox-example']/button");
+    public final By TEXT_AFTER_CLICK_ENABLE = By.xpath("/html/body/div[2]/div/div[1]/form[2]/p");
+    public final By TEXT_AFTER_BUTTON = By.id("message");
+    public final boolean EXP_RESULT_CHECKBOX = false;
+    public final boolean EXP_RESULT_INPUTFIELD = true;
+    public final String EXP_RESULT_MESSAGE_CHECKBOX = "It's gone!";
+    public final String EXP_RESULT_MESSAGE_INPUTFIELD = "It's enabled!";
```

Comment:
1 ) expected results should not be declared in Page classes, they are considered as Test data
2 ) not necessary to declare constants for boolean values